requests.json

```diff
@@ -55,5 +64,14 @@
                 "schema": { "$ref": "./schemas.json#/Sitelink" }
             }
         }
+    },
+    "Term": {
+        "description": "An object that contains a language : string pair",
```

Comment:
It's more like "an object that contains a `value` and the information in which `language` that value is in".

Reply:
I know. couldn't find a better description.